.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - phpunit-version: 4
```

Comment:
we claim `^4.8.36` , not ^4. can we narrow it down, here?

Reply:
`composer update phpunit/phpunit:^4` will respect both constraint from `composer.json` and the temporary one (in this case `^4 && ^4.8.36`).